data/civ1/buildings.ruleset

```diff
@@ -336,12 +336,12 @@ The amount of stored food will be set to half full whenever a city\
 ; In Civ1, city size reduction does not generate food like this.
 ; Dare I ask where this food comes from?? :-)
 _("\
-Famine can occur when food stock reaches zero due to a food deficit\
- (which a Granary merely delays), in which case it leads to the loss\
+Famine can occur when food stock reaches zero due to a food deficit,\
+ which a Granary merely delays. In which case it leads to the loss\
```

Comment:
While not incorrect, using a full stop before `in which case` seems to be far less common than using a comma.